res/wrap.jl

```diff
@@ -0,0 +1,46 @@
+using Clang
+
+if length(ARGS) != 1
+  error("Pass the path to the Flux headers as the first argument")
+end
+
+header_dir = ARGS[1]
+isdir(header_dir) || error("$header_dir does not exist")
+
+const FLUX_INCLUDE = normpath(header_dir)
+
+# Fails with `include/flux/core/barrier.h:24:1: error: unknown type name 'flux_future_t'`
+# as far as I can tell `barrier.h` is not including `future.h`
+# headers = ["flux/core.h"]
+# core_headers = readdir(joinpath(FLUX_INCLUDE, "flux/core"))
+# filter!(h->endswith(h, ".h"), core_headers)
+# core_headers = map(h->joinpath("flux/core", h), core_headers)
+# append!(headers, core_headers)
+
+# Flux comes with preprocessed bindings for Python
+# they still come with some need for fixups
```

Comment:
Are the fixups below being applied?  Or were you able to make do with the preprocessed python headers?

Reply:
Yes I had to apply these fixups manually